docs/explanation/overlays.rst

```diff
@@ -0,0 +1,176 @@
+****************
+``OVERLAY`` Step
+****************
+
+It is sometimes necessary to modify the base filesystem when generating a
```

Comment:
I guess what I'm missing here is some kind of definition of "base filesystem" here. Maybe a distinction between this "base" filesystem and the build filesystem where the lifecycle runs?

Reply:
I've incorporated your suggestion from Sergio's comment above, which should fix both :-)